<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProduct</class>
 <widget class="QDialog" name="AddProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lägg till Produkt</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ProductNameLabel">
          <property name="text">
           <string>Produkt namn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="productName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="IdNrLabel">
          <property name="text">
           <string>Serie nummer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="IdNrText"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="LevLabel">
          <property name="text">
           <string>Leverantör</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Lev"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="capicityLabel">
          <property name="text">
           <string>Lager kapacitet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="capicety"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="pricePerItemLabel">
          <property name="text">
           <string>Pris per vara</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pricePerItem"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="NrOfItemsLabel">
        <property name="text">
         <string>Antal varor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NrOfItems"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddProduct</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddProduct</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
